### ISSUE TITLE
Is this code commercially available?

### ISSUE THREAD

**Issue by dgrid** [qwymierne/semantic_soft_segmentation#2]
I want to use your codes, but no information about license is written.
Can I use your code commercially?

**Comment by qwymierne** [comment]
Yes, You can use code in any way You like. The project is definitely far from finish, but probably I won't improve it any more. It is public by mistake, but I'm not going to make it private right now.

**Comment by qwymierne** [comment]
Yes, you can use the code in any way you like. The project is far from the finish, but probably I won't improve it any more. It is public by mistake, but I'm not going to make it private right now.
